### PR TITLE
Setup skill selector

### LINKED ISSUE #28: Implement skill selector

### PR DESCRIPTION
**Pull request by BenAlderfer**
Fixes #28 

Still missing lots of skill data and icons, but its working

### CODE REVIEW

File: src/components/shared/data/skills.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    "Shield": [
+        {
+            name: "Shield",
+            baseStats: {
+                Cooldown: "1s",
+            },
+            tiers: {
+                1: {
+
+                },
```
Comment (Michael-Longley) [inline src/components/shared/data/skills.js:321]: Are these all partial data files or is this intentional? What does an empty tier mean?

File: src/components/shared/card.js [hunk 2]
```diff
@@ -384,12 +426,12 @@ export function ArmorCard(props) {
     )
 }
 
-ArmorCard.propTypes={
+ArmorCard.propTypes = {
```
Comment (Michael-Longley) [inline src/components/shared/card.js:429]: This has really necessitated a style file for our IDEs. We should set up ESLINT. I think that should fix this but not certain.

**Review by Michael-Longley — APPROVED** [review]
Had one question otherwise LGTM

**Comment by JakeBash** [comment]
My comments mimic that of Mike's. Other than that, looks solid to me

**Review by JakeBash — APPROVED** [review]

File: src/components/shared/data/skills.js (hunk 1 above)
Comment (BenAlderfer) [reply src/components/shared/data/skills.js:321]: The point is to layout the structure. Like I was saying there’s a lot of data entry still left

**Pull request merged**